pkg/microservice/user/core/handler/user/user.go

```diff
@@ -1,14 +1,86 @@
 package user
 
 import (
+	"fmt"
+
+	"github.com/dexidp/dex/connector/ldap"
 	"github.com/gin-gonic/gin"
+	"go.uber.org/zap"
+
+	ldapv3 "github.com/go-ldap/ldap/v3"
 
+	"github.com/koderover/zadig/pkg/microservice/systemconfig/core/service"
```

Comment:
don't import another service directly, if you need the structs, define it again in the client or in your service